fitbit-react/src/Pages/Homepage/FeaturedProducts/FeaturedProductsCard.js

```diff
@@ -1,42 +1,37 @@
 import React from "react";
 import { BsFillHeartFill } from "../../../Assets/icons";
-function FeaturedProductsCard({
-  imageUrl,
-  cardTitle,
-  price,
-  discountedPrice,
-  discountPercent,
-  redirectUrl,
-}) {
+import { Link } from "react-router-dom";
+
+function FeaturedProductsCard({ product, redirectUrl = "/products" }) {
```

Comment:
Can destructure product in the top, so you won't have to prefix product. everywhere in this file

Reply:
Done